pokerole_irl/user_app/models.py

```diff
@@ -11,3 +11,25 @@ class Profile(models.Model):
 
     def __str__(self):
         return self.user.username
+
+
+class Setting(models.Model):
+    owner = models.ForeignKey(User, on_delete=models.CASCADE)
+    members = models.ManyToManyField(User, related_name='+', blank=True)
```

Comment:
The related name is the name of the attribute of this field in the other direction: Calling `User.objects.first().+` will give a queryset of all the settings the first user is a member of. It would probably be good to set the related_name to "settings". On this note, maybe it is not a good idea to have two keys to users. Maybe having the "members" relation is enough, if users are related in a "through"-table.

For example, the relation between User and Setting could be a Membership object with three attributes: user (a user object), setting (a Setting object) and position (A string choice, with the choices "member" and "owner"). This is highly flexible, and would also allow more positions in the future ("game master" for example).

Reply:
Correct me if I'm wrong, but related_name=+ should result in no backwards relation in the users table. I did it to avoid conflicts. Is it worth to construct a more complicated relation object?

Reply:
Relation is now done through a separate model.